### DISCUSSION TITLE
epub format

### DISCUSSION THREAD

**Discussion by coke** [perl6/doc#4142] [Ideas]
As part of the generation of the website, a single-page HTML file was historically created to provide users with a version of the site that could be read offline.

Instead, we should generate a epub file to fill this niche and remove the static html file.

**Comment by 2colours** [comment]
Yes, agreed. Is there a plan on the workflow of that?

**Comment by 2colours** [comment]
This idea will be manifested in https://github.com/Raku/doc-website/issues/75

**Comment by finanalyst** [comment]
Does anyone here (/in the community) have any experience of generating epub from html?
From a cursory glance, epub content is a zipped version of a static html tree, but the files are given as xhtml. And there are two extra content and config files. 
1. Is the xhtml extra functionality actually used?
2. Is there anything to beware of in generating the config files?

Edit: Found epub [v3.2 specification ](https://www.w3.org/publishing/epub3/epub-spec.html)

Unless someone has an objection, this is the standard to be targeted.
